orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/MessageSpec.kt

```diff
@@ -0,0 +1,48 @@
+package com.netflix.spinnaker.orca.q
+
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.context
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+import org.junit.Assert
+
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+object MessageSpec : Spek({
```

Comment:
I like this a lot. I'm not sure you can get rid of the `::class.java`, but this interface should work perfectly for us - much better experience than what I proposed.